mirrulations-client/src/mirrclient/client.py

```diff
@@ -129,6 +129,38 @@ def get_job(self):
 
         return job
 
+    def process_job_results(self, job, job_result):
+        # TODO new function for handling errors
+        # This function is not complete yet, but should replace send_job
+        data = {
+            'job_type': job['job_type'],
+            'job_id': job['job_id'],
+            'results': job_result,
+            'reg_id': job['reg_id'],
+            'agency': job['agency']
+        }
+        print(f'Sending Job {job["job_id"]} to Work Server')
+        if 'error' in job_result:
+            # Handles errors in job_results
+            result = self.redis.hget('jobs_in_progress', job.get('job_id'))
+            self.redis.hset('invalid_jobs', job.get('job_id'), result)
+            
+        data['directory'] = self.path_generator.get_path(job_result)
+
+        self._put_results(data)
+        # We should increment the redis counter for the type downloaded here 
+
+        comment_has_attachment = self.does_comment_have_attachment(job_result)
+        json_has_file_format = self._document_has_file_formats(job_result)
+
+        if data["job_type"] == "comments" and comment_has_attachment:
+            self.download_all_attachments_from_comment(data, job_result)
+        if data["job_type"] == "documents" and json_has_file_format:
+            document_htm = self._get_document_htm(job_result)
+            if document_htm is not None:
+                self.download_htm(job_result)
+        
```

Comment:
Wondering if we could move this functionality into a different function maybe `_put_results`